studio/components/layouts/AccountLayout/WithSidebar.tsx

```diff
@@ -86,8 +86,8 @@ const WithSidebar: FC<Props> = ({
 export default WithSidebar
 
 const LinksWithHeaders: FC<any> = ({ links, subitems, subitemsParentKey }) => {
-  return links.map((x: any) => (
-    <div key={x.heading} className="dark:border-dark border-b py-5 px-6">
+  return links.map((x: any, i: number) => (
+    <div key={x.heading || `${i}`} className="dark:border-dark border-b py-5 px-6">
```

Comment:
any reason why we need to do this? is there a link that has no heading?

Reply:
This was just to remove an annoying error in the console. I think there must have been a link with no heading causing duplicate key errors. This was just a quick fix.
Happy to investigate further if you think there's a benefit/cleaner code?

Reply:
ahh okay no worries i can get into this while i test